Countries/ListViewController.swift

```diff
@@ -55,9 +66,16 @@ class ListViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return TableConstants.heightForRow
     }
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        
+        let header = UITableViewHeaderFooterView()
+        header.addSubview(headerView)
+        return header
```

Comment:
Should it have some additional constraints or frame setting?

Reply:
story, it should, but I've just changed it in other branch for header